images/router/haproxy/conf/haproxy-config.template

```diff
@@ -22,6 +22,9 @@
 {{/* cidrListPattern: Match a space separated list of CIDRs; e.g. 192.168.21.23/24 192.10.2.12 */}}
 {{- $cidrListPattern := printf `(?:%s(?: +%s)*)` $cidrPattern $cidrPattern -}}
 
+{{/* cookie name pattern: */}}
+{{- $cookieName := "^[a-zA-Z0-9_-][a-zA-Z0-9_-]*" -}}
```

Comment:
No need for the leading ^ since our functions add ^(...)$ around the provided regexes.

Can you name the variable $cookieNameRE to match the others

Reply:
OK